# Relative Roll Estimate Message Structure

Header header

# Relative Roll Angle (deg)
float64 relative_roll